content/applications/hr/payroll/contracts.rst

```diff
@@ -3,223 +3,360 @@ Contracts
+from the top menu. All employee contracts are displayed in a default list view, grouped in four
+categories: :guilabel:`New`, :guilabel:`Running`, :guilabel:`Expired`, and :guilabel:`Cancelled`.
+
+The default view has all categories collapsed. The number of contracts in each category is displayed
+next to the category name (example: `Running (20)`). To view the list of contracts under any
+category, click on the category and the list expands downward, showing all contracts beneath it.
 
 .. image:: contracts/contracts-overview.png
    :align: center
    :alt: Contracts dashboard view showing running contracts and contracts with issues.
 
 .. note::
    The list of contracts in the *Payroll* application matches the list of contracts in the
-   *Employees* application. The default contracts view in the *Payroll* application displays running
-   contracts and contracts needing attention, while the default contracts view in the *Employees*
-   application displays all contracts in a kanban view, organized by their stage, regardless of
-   status. All contracts can be viewed by changing the filters.
+   *Employees* application.
 
 Create a new contract
 =====================
 
-In order for an employee to be paid, an active contract is required. If a new contract is needed,
-click the :guilabel:`Create` button on the contracts dashboard. A contract form appears where the
-information can be entered. Required fields are underlined in bold.
+In order for an employee to be paid, an active contract is required. When a new contract is needed,
+click the :guilabel:`New` button on the contract dashboard. A contract form appears where the
+information can be entered.
 
 Required fields
 ---------------
 
+- :guilabel:`Contact Reference`: type in the name or title for the contract, such as `John Smith
+  Contract`.
+- :guilabel:`Contract Start Date`: the date the contract starts. To choose a date, click the
+  drop-down menu, navigate to the correct month and year with the :guilabel:`< > (arrow)` icons,
+  then click on the :guilabel:`date`.
+- :guilabel:`Working Schedule`: select one of the working schedules from the drop-down menu.
+
+.. tip::
+   The :guilabel:`Working Schedule` drop-down menu displays all the working schedules for the
+   selected :guilabel:`Company`. To modify or add to this list, go to :menuselection:`Payroll app
+   --> Configuration --> Working Schedules`. Click :guilabel:`New` and create a new working
+   schedule, or click on an existing working schedule and make edits.
+
+- :guilabel:`Work Entry Source`: select how the :doc:`work entries <../../hr/payroll/work_entries>`
+  are generated. Click the radio button next to the desired selection. The options are:
+
+  - :guilabel:`Working Schedule`: work entries are generated based on the selected
+    :guilabel:`Working Schedule`.
+  - :guilabel:`Attendances`: work entries are generated based on the employee's attendances logged
```

Comment:
I agree.  I used "check in records" since that was the best I could come up with that was closest to what they are called. They are called attendance records in the attendances app- because of course they are  =D